src/components/DateTimePicker/DateTimePicker.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Picker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+const DateTimePicker = props => {
+  const { date, setDate, customInput, customInputProps, isTimeOnly, isSmallScreen, ...rest } = props;
```

Comment:
well, it's a function that's gonna set the date. it will end up inside the onChange handler, but it is not the entire handler itself. I still think setDate is a predictable title, let's discuss this?